client/src/boot/registerRoutes.js

```diff
@@ -0,0 +1,14 @@
+/* global reactAdminConfig */
+import reactRouteRegister from 'lib/ReactRouteRegister';
+import Config from 'lib/Config';
+import ReactAdminBoostrap from 'components/ReactAdminBoostrap';
+
+export default () => {
+  const sectionConfig = Config.getSection(reactAdminConfig.configKey);
+
+  reactRouteRegister.add({
+    path: `/${sectionConfig.url}`,
+    component: ReactAdminBoostrap,
+    exact: false
+  });
+};
```

Comment:
This defines a catch-all route. `<ReactAdminBoostrap />` will figure out how to load your custom component.